packages/server/src/app.ts

```diff
@@ -204,20 +204,20 @@ export function initAppServices(config: MedplumServerConfig): Promise<void> {
 }
 
 export async function shutdownApp(): Promise<void> {
-  cleanupHeartbeat();
   await closeWebSockets();
-  await closeWorkers();
-  await closeDatabase();
-  await closeRedis();
-  closeRateLimiter();
-
   if (server) {
     await new Promise((resolve) => {
       (server as http.Server).close(resolve);
     });
     server = undefined;
   }
 
+  cleanupHeartbeat();
```

Comment:
I think this one might need to still be done before `closeWebSockets()` since the heartbeat action is usually dependent on websockets being open.